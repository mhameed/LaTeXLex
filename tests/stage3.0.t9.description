author: ehc
date: 09/04/08
description: Tests preservation, maths and verbatim environments inside others