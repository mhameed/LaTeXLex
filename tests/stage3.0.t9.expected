\begin{itemize}
\item Here is an itemize that should not be touched
\item And an item that contains some maths:
x^2 + y^2 = z^2
\item and an item that contains an itemize:
\begin{itemize}
\item Internal items
\item and another which contains a verbatim
\begin{verbatim}
Some
stuff
\end{verbatim}
\end{itemize}

\item and an item with some spaces at the end   
\end{itemize}

And some more stuff that should be pulled up.
